internal/transport/http2_client.go

```diff
@@ -1023,6 +1023,13 @@ func (t *http2Client) Close(err error) {
 	case <-timer.C:
 		t.logger.Infof("Failed to write a GOAWAY frame as part of connection close after %s. Giving up and closing the transport.", goAwayLoopyWriterTimeout)
 	}
+	select {
```

Comment:
this shouldn't be here either :). Probably checkout to main branch and pull upstream again and then create a fresh branch for it

```
git checkout master
git pull upstream master
git rebase upstream/master
git checkout -b <new-branch>
```